README.md

```diff
@@ -351,14 +392,29 @@ data conversion can be found here:
 The following steps can help diagnose common issues encountered while running
 HarbourBridge.
 
-### 1. Verify driver configuration
+### 1. Verify source profile configuration
+
+First, check that the source profile is correctly configured to connect to your
+database. Source profile configuration varies depending on the database.
+
+#### 1.1 Direct access to PostgreSQL
+
+See [Directly connecting to a PostgreSQL database](sources/postgres/README.md#directly-connecting-to-a-postgresql-database) for troubleshooting direct access to PostgreSQL.
+
+#### 1.2 Direct access to MySQL
+
+See [Directly connecting to a MySQL database](sources/mysql/README.md#directly-connecting-to-a-mysql-database) for troubleshooting direct access to MySQL.
+
+#### 1.3 Direct access to DynamoDB
 
-First, check that the driver is correctly configured to connect to your
-database. Driver configuration varies depending on the driver used.
+See [DynamoDB example usage](sources/dynamodb/README.md#example-dynamodb-usage)
+for troubleshooting direct access to DynamoDB.
 
-#### 1.1 pg_dump
+### 2. Generating dump files
 
-If you are using pg_dump (-driver=pg_dump), check that pg_dump is
+#### 2.1 Generating pg_dump file
+
+If you are using pg_dump , check that pg_dump is
```

Comment:
It would make sense to move this into the source specific readme

Reply:
I agree, but would ask to take this clean up in a separate PR, such that this PR can be wrapped up.